### PR TITLE
qtile: add page

### PR DESCRIPTION
**Pull request by Rolv-Apneseth**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** `v0.22.1`

Added a page for the `qtile` tiling window manager for Linux with some examples of useful commands

### CODE REVIEW

File: pages/linux/qtile.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# qtile
+
+> A full-featured, hackable tiling window manager written and configured in Python.
+> More information: <https://docs.qtile.org/en/latest/manual/commands/shell/index.html>.
+
+- Start the window manager, if it is not running already (should ideally be run from `.xsession` or similar):
+
+`qtile start`
+
+- Check a configuration file for any compilation errors (default location is  `~/.config/qtile/config.py`):
```
Comment (pixelcmtd) [inline pages/linux/qtile.md:10]:
```suggestion
- Check the configuration file for any compilation errors (default location is  `~/.config/qtile/config.py`):
```

File: pages/linux/qtile.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+`qtile check`
+
+- Show current resource usage information:
+
+`qtile top --force`
```
Comment (pixelcmtd) [inline pages/linux/qtile.md:16]: Why does this need `--force`?

File: pages/linux/qtile.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+- Open the program `xterm` as a floating window on the group named `test-group`:
+
+`qtile run-cmd -g {{test-group}} -f {{xterm}}`
+
+- Restart the window manager:
+
+`qtile cmd-obj -o cmd -f restart`
```
Comment (pixelcmtd) [inline pages/linux/qtile.md:24]:
```suggestion
`qtile cmd-obj --object cmd --function restart`
```

File: pages/linux/qtile.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+`qtile check`
+
+- Show current resource usage information:
+
+`qtile top --force`
+
+- Open the program `xterm` as a floating window on the group named `test-group`:
+
+`qtile run-cmd -g {{test-group}} -f {{xterm}}`
```
Comment (pixelcmtd) [inline pages/linux/qtile.md:20]:
```suggestion
`qtile run-cmd --group {{test-group}} --float {{xterm}}`
```

File: pages/linux/qtile.md (hunk 2 above)
Comment (Rolv-Apneseth) [reply pages/linux/qtile.md:16]: When run without `--force` (for the first time in a session) the following error message is output

```
tracemalloc not started on qtile, start by setting PYTHONTRACEMALLOC=1 before starting qtile
or force start tracemalloc now, but you'll lose early traces
```

Would this be ok to leave the command as `qtile top`? Not sure what the protocol is here, sorry

Comment (pixelcmtd) [reply pages/linux/qtile.md:16]: Ok, that's sensible.

File: pages/linux/qtile.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
+# qtile
+
+> A full-featured, hackable tiling window manager written and configured in Python.
```
Comment (pixelcmtd) [inline pages/linux/qtile.md:3]:
```suggestion
> A fully-featured, hackable tiling window manager written and configured in Python.
```

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**